### PR TITLE
pointerEvents by props

### PR DESCRIPTION
**Pull request by skinet2009**
Added pointerEvents to props, to make possible adding component with touchable element

### CODE REVIEW

**Comment by corentin-gautier** [comment]
@crazycodeboy could you merge this ?

**Comment by FredericoVieira** [comment]
Please do this merge! I'm need this feature!

**Comment by olfek** [comment]
hello @crazycodeboy

**Comment by olfek** [comment]
@skinet2009 conflicts need resolving

**Comment by skinet2009** [comment]
> @skinet2009 conflicts need resolving

ok, I will fix it soon

**Comment by olfek** [comment]
this is ready for merge @crazycodeboy

File: index.js [hunk 1]
```diff
@@ -167,5 +168,6 @@ Toast.defaultProps = {
     fadeInDuration: 500,
     fadeOutDuration: 500,
     opacity: 1,
+    pointerEvents: 'none',
```
Comment (olfek) [inline index.js:171]: Should this have been `auto` as it was the old default value? @crazycodeboy @skinet2009

**Pull request merged**